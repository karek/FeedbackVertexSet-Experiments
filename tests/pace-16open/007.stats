36,239,26
