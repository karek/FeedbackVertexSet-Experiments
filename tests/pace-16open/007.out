17
3 22 33 8 0 18 29 32 12 19 10 28 4 5 17 27 9 
